janus_core/helpers/mlip_calculators.py

```diff
@@ -7,6 +7,7 @@
 """
 
 from ase.calculators.calculator import Calculator
+import torch
```

Comment:
From speaking to @oerc0122, is this definitely slower, since it still needs to evaluate the init either way?

I only changed it since pylint doesn't like `from torch import float32`, so happy to change it back with a warning disable if you definitely think it makes a difference

Reply:
always looked to me but is fine